Fixes etc

WibbenZ opened the pull request:
﻿Fixed all Shimmer ~ items, #84
Fixed special foods, the "drowning" function still has to be added, we could modify the drowning.lua script, but I don't feel like that is the best way tbh.
Closes #43
Fixed the "random spawns" from #56
Renamed data/lib to data/library.
Added a template for the world board.
Fixed most POI bosses how ever the loot should be checked.
Fixed the following bugs with the postman quest.
  * Item name (itemid: 2332).
  * Quest log string.
  * Lower case in a message in Kevin.lua.
  * Added a script to set the correct storage for the mission where you should get Waldo's posthorn.
Fixed Empror Rehal greeting message.
Fixed most of the bugs in The Hidden City of Beregar quest.

WibbenZ commented:
Well got it working, but looks as if I did something wrong? @ninjalulz

ninjalulz commented:
You should have updated your master branch before you pushed changes to your repository.

```
git pull upstream master
git push
```

Work in branches, saves you a lot of trouble.
```
git branch newBranch
git checkout branchName
-- add changes
git push origin branchName
```

WibbenZ commented:
Going insane soon...
This is what I did..
git clone page
I got the latest files in my GitHub folder.
Added all the things I had modified and then tried to upload it with gitshell but that did not work, when I tried it with the git software~ it worked just fine. Except the conflicts.
My master branch is that the one that I have like github.com/wibbenz/server?
If that is the case how can I do it now? The commands you wrote does not work.
![](http://4.ii.gl/hriH9sken.png)
Even tried to download the latest code and add the protected folder git and see if it would work, it diden't.
